codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2ConnectionDecoder.java

```diff
@@ -135,6 +153,59 @@ public Http2Settings localSettings() {
         return settings;
     }
 
+    /**
+     * Applies settings sent from the local endpoint.
+     */
+    private void applyLocalSettings(Http2Settings settings) throws Http2Exception {
```

Comment:
this method was moved in the file ... it doesn't look like anything changed, is that right?